src/server.jl

```diff
@@ -66,6 +66,11 @@ end
 function _exeflags_and_env(options)
     exeflags = map(String, options["format"]["metadata"]["julia"]["exeflags"])
     env = map(String, options["format"]["metadata"]["julia"]["env"])
+    # Use `--project=@.` if neither `JULIA_PROJECT=...` nor `--project=...` are specified
+    if !any(Base.Fix2(startswith, "JULIA_PROJECT="), env) &&
+       !any(Base.Fix2(startswith, "--project="), exeflags)
```

Comment:
```
julia> startswith("JULIA_PROJECT=")
(::Base.Fix2{typeof(startswith), String}) (generic function with 1 method)
```

works in 1.6+, so no need for manually calling `Fix2` unless I'm missing something here @devmotion?

Reply:
Ah I missed that the one-argument method exists 👍